core/src/test/java/org/infinispan/api/CacheAPITest.java

```diff
@@ -17,18 +17,18 @@
 import java.util.Map.Entry;
 import java.util.Set;
 
-import jakarta.transaction.TransactionManager;
-
 import org.infinispan.commons.util.CloseableIterator;
 import org.infinispan.configuration.cache.CacheMode;
 import org.infinispan.configuration.cache.Configuration;
 import org.infinispan.configuration.cache.ConfigurationBuilder;
+import org.infinispan.configuration.cache.IsolationLevel;
 import org.infinispan.manager.EmbeddedCacheManager;
 import org.infinispan.test.TestingUtil;
 import org.infinispan.test.fwk.TestCacheManagerFactory;
-import org.infinispan.configuration.cache.IsolationLevel;
 import org.testng.annotations.Test;
 
+import jakarta.transaction.TransactionManager;
+
```

Comment:
Unrelated change, can be reverted.

Reply:
I have "organize imports" as a save action :)